examples/all-clusters-app/all-clusters-common/src/bridged-actions-stub.cpp

```diff
@@ -129,3 +134,12 @@ Status ActionsDelegateImpl::HandleDisableActionWithDuration(uint16_t actionId, u
     // Not implemented
     return Status::NotFound;
 }
+
+void emberAfActionsClusterInitCallback(EndpointId endpoint)
+{
+    VerifyOrReturn(endpoint == 1);
+    VerifyOrReturn(sActionsDelegateImpl == nullptr && sActionsServer == nullptr);
+    sActionsDelegateImpl = new Actions::ActionsDelegateImpl;
```

Comment:
would there be a way to ensure leak analysis does not report these as leaks? Maybe using std::unique_ptr?

Reply:
Done